src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/ReadInputArgumentCollection.java

```diff
@@ -37,6 +38,11 @@
               optional = true)
     protected List<String> readIndices;
 
+    @Advanced
+    @Argument(fullName = "reads-look-ahead-window-size", shortName="reads-look-ahead-window-size",
+                doc = "Window size for reads look-ahead query caching")
+    protected int readsLookaheadWindowSize = 100 * 1024;
```

Comment:
I don't think that this should be exposed to the user, but being similar to the feature-cache: it should be choosen by the walker-class what is the best value.